common/app/common/metrics.scala

```diff
@@ -165,6 +165,17 @@ object ApplicationMetrics {
   def apply(metrics: FrontendMetric*): ApplicationMetrics = ApplicationMetrics(metrics.toList)
 }
 
+object RenderingMetrics {
```

Comment:
Is it worth being a bit more explicit that this is for article rendering somewhere?